app/Views/GlobalGui.js

```diff
@@ -52,8 +52,133 @@ define(['backbone', 'JST', 'q', 'bootstrap', 'app'], function(Backbone, template
                 app.growl('Launching');
             });
 
+            // sketch of approach
+
+            // for right now, setting the APM severity to have a one-to-one with growl type
+            // growl types:
+            //      warning, success, danger, error
+            var SEVERITY_HIGH = 'danger';
+            var SEVERITY_MEDIUM = 'warning';
+            var SEVERITY_LOW = 'success';
+
+            // these were pulled from the APM source wherever "gcs_send_text_P" was used to send a STATUSTEXT
+            var statustextRegex = {
+                "ARMING MOTORS" : {severity: SEVERITY_HIGH},
```

Comment:
I saw multiple ARMING MOTORS growls pop up.  I think we may also need to debounce all Growls over (say) 5 seconds -- so a single message won't get repeated unless it's really happening often.